contact_updater/templates/agency_form.html

```diff
@@ -0,0 +1,72 @@
+{% extends "base.html" %}
+{% block head %}
+    {{super()}}
+<script type="text/javascript">
+    function getContent(){
```

Comment:
Made contenteditables work by replacing each invisible form's value with its corresponding contenteditable innerHTML. It's a little messy so if it turns out we don't want to move down this path we can always roll back.